diff --git a/Core/CapslockX-Core.ahk b/Core/CapslockX-Core.ahk
index 942861d..c467a1a 100644
--- a/Core/CapslockX-Core.ahk
+++ b/Core/CapslockX-Core.ahk
@@ -355,12 +355,13 @@ CLX_Up()
         } else {
             ; 单击 CapsLockX
             if (CLX_上次触发键 == "CapsLock") {
+                Send {Blind}{Esc} ; 发送 Esc 键
                 ; 切换 CapsLock 状态（原功能）
-                if (GetKeyState("CapsLock", "T")) {
-                    SetCapsLockState, Off
-                } else {
-                    SetCapsLockState, On
-                }
+                ; if (GetKeyState("CapsLock", "T")) {
+                ;     SetCapsLockState, Off
+                ; } else {
+                ;     SetCapsLockState, On
+                ; }
             }
             ; 单击 空格键
             if (CLX_上次触发键 == "Space") {
